sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -94,14 +94,14 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/Bensuo/unified-runtime.git")
   # commit ebf873fb5996c9ddca32bbb7c9330d3ffe15473c
   # Merge: 633ec408 8f375039
   # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
   # Date:   Thu May 2 10:56:55 2024 +0100
   #    Merge pull request #1535 from przemektmalon/przemek/sampled-image-fetch
   #    [Bindless][Exp] Add device queries for sampled image fetch
-  set(UNIFIED_RUNTIME_TAG ebf873fb5996c9ddca32bbb7c9330d3ffe15473c)
+  set(UNIFIED_RUNTIME_TAG 8438b8a1de1ddc6e05e3bc012fb6be74d2e830e4)
```

Comment:
https://github.com/oneapi-src/unified-runtime/pull/1499 has been updated please update these varaibles as suggested.

Also, please change apply the diff below so all the adatpers are synchronized on the same tag.

```diff
   fetch_adapter_source(level_zero
     ${UNIFIED_RUNTIME_REPO}
-    # commit fb342f06e1ac244c5995b7be458d0c32a1ba17ab
-    # Merge: a44e81b5 7186d6ce
-    # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-    # Date:   Fri May 3 11:55:26 2024 +0100
-    #     Merge pull request #1549 from igchor/event_fix
-    #     Fix adding event to queue cache
-    fb342f06e1ac244c5995b7be458d0c32a1ba17ab
+    ${UNIFIED_RUNTIME_TAG}
   )
```

Once that's updated, make this ready to review and I'll approve the changes.